app/Auth/Authorization.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Auth;
+
+use App\Legacy\Legacy;
+use App\Models\Logs;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Session;
+use RuntimeException;
+
+class Authorization extends Auth
+{
+	public static function id(): int|null
+	{
+		return is_int(parent::id()) ? parent::id() : null;
+	}
+
+	public static function isAdmin(): bool
+	{
+		return parent::user()?->isAdmin() === true;
```

Comment:
```suggestion
		return parent::user()?->isAdmin();
```

`isAmin` already returns a boolean.

Reply:
No, cf earlier comment.